tests/compile-fail/arithmetic.rs

```diff
@@ -0,0 +1,31 @@
+#![feature(plugin)]
+#![plugin(clippy)]
+
+#![deny(integer_arithmetic, float_arithmetic)]
+#![allow(unused, shadow_reuse, shadow_unrelated, no_effect)]
+fn main() {
+    let i = 1i32;
+    1 + i; //~ERROR integer arithmetic detected
+    i * 2; //~ERROR integer arithmetic detected
+    1 % //~ERROR integer arithmetic detected
```

Comment:
Ah ok. That, here, is confusing, maybe add a regular comment that explains that and/or put that in another section. Also `i / 2` should probably not lint, it does not panic ever.